### PR TITLE
[VL] Refine CMAKE_CXX_FLAGS setting logic

### PR DESCRIPTION
**Pull request by Yohahaha**
## What changes were proposed in this pull request?

Modify CMAKE_CXX_FLAGS generation logic for each native module, and make it clearly to see all the CMAKE_CXX_FLAGS.

### CODE REVIEW

**Comment by FelixYBW** [comment]
Is there impact to build process?

**Comment by Yohahaha** [comment]
> Is there impact to build process?

no changes, just a little refactor.

**Comment by Yohahaha** [comment]
@marin-ma @PHILO-HE please help take a look, thank you!

File: cpp/velox/CMakeLists.txt [hunk 1]
```diff
@@ -52,6 +47,19 @@ if (NOT DEFINED VELOX_HOME)
   message(STATUS "Set VELOX_HOME to ${VELOX_HOME}")
 endif()
 
+execute_process(
```
Comment (marin-ma) [inline cpp/velox/CMakeLists.txt:50]: What is this change for? Could you add some notes here?

Comment (Yohahaha) [reply cpp/velox/CMakeLists.txt:51]: keep cpp/velox module's compile option same as Velox.

**Review by marin-ma — APPROVED** [review]
Thanks. @PHILO-HE Do you have any comments?

**Review by PHILO-HE — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**